valhalla/baldr/graphconstants.h

```diff
@@ -364,6 +364,12 @@ enum class TaggedValue : uint8_t { // must start at 1 due to nulls
   // see https://github.com/valhalla/valhalla/issues/3262
   kTunnel = static_cast<uint8_t>('1'),
   kBridge = static_cast<uint8_t>('2'),
```

Comment:
these were a bug and thus casted from char to int. you should start counting from int=2

Reply:
thank you @kevinkreiser